src/modules/aws-ebs/sub-modules/snapshots/copySnapshotToTargetRegion.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as AWS from "aws-sdk";
+import * as AWSMock from "aws-sdk-mock";
+import * as sinon from "sinon";
+import * as buildCopySnapshotDescription from "./buildCopySnapshotDescription";
+import { copySnapshotToTargetRegion } from "./copySnapshotToTargetRegion";
+
+AWSMock.setSDKInstance(AWS);
+
+describe("copySnapshotToTargetRegion()", () => {
+  afterEach(() => {
+    AWSMock.restore("EC2");
+    sinon.restore();
+  });
+
+  it("should return SnapshotId when copySnapshot call is successful", async () => {
```

Comment:
For this test, I think the best thing to test is if the mocks are being called with the right parameters. Like the EC2 constructor receiving the `quarantineAwsRegion` as opposed to `sourceAwsRegion`.